config/sidekiq_scheduler.yml

```diff
@@ -0,0 +1,5 @@
+transmit_education_spool_files:
+  cron: "0 3 * * * America/New_York"
+  class: EducationForm::CreateDailySpoolFiles
+  queue: default
```

Comment:
Also, why call the section transmit_education_spool_files while the job is called CreateDailySpoolFiles?